src/module/item/weapon/document.ts

```diff
@@ -353,6 +353,15 @@ class WeaponPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Ph
             traits.value.push("unarmed");
         }
 
+        // Ensure handwraps have otherTags set
+        if (
+            this.system.category === "unarmed" &&
```

Comment:
Do you think it might be safer to omit this? It looks like even non-unarmed handwraps still acted as handwraps before, and this might break it for people who tinkered with the details.